nms/app/packages/magmalte/app/views/network/FEGServicingAccessGatewayTable.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {
+  feg_lte_network,
+  gateway_id,
+  gateway_name,
+  network_id,
+  network_name,
+} from '@fbcnms/magma-api';
+
+import ActionTable from '../../components/ActionTable';
+import Link from '@material-ui/core/Link';
+import LoadingFiller from '@fbcnms/ui/components/LoadingFiller';
+import React, {useEffect, useState} from 'react';
+import nullthrows from '@fbcnms/util/nullthrows';
+
+import {FetchGateways} from '../../state/lte/EquipmentState';
+import {getServicedAccessNetworks} from '../../components/ServicingAccessGatewayKPIs';
+import {useEnqueueSnackbar} from '@fbcnms/ui/hooks/useSnackbar';
+import {useRouter} from '@fbcnms/ui/hooks';
+
+type ServicingAccessGatewayRowType = {
+  networkId: network_id,
+  networkName: network_name,
+  gatewayId: gateway_id,
+  gatewayName: gateway_name,
+};
+
+/**
+ * Returns a table consisiting of the serviced access gateways alongside
+ * the serviced network in which they are under.
+ */
+export default function ServicingAccessGatewayInfo() {
+  const {match} = useRouter();
+  const enqueueSnackbar = useEnqueueSnackbar();
+  const networkId: string = nullthrows(match.params.networkId);
+  const [servicedAccessGatewaysInfo, setServicedAccessGatewaysInfo] = useState<
+    Array<ServicingAccessGatewayRowType>,
+  >([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const getServicedAccessGatewaysInfo = async (
+      servicedAccessNetworks: Array<feg_lte_network>,
+    ): Promise<Array<ServicingAccessGatewayRowType>> => {
+      const newServicedAccessGatewaysInfo = [];
+      for (const servicedAccessNetwork of servicedAccessNetworks) {
+        const servicedAccessGateways = await FetchGateways({
+          networkId: servicedAccessNetwork.id,
+          undefined,
+          enqueueSnackbar,
+        });
+        //Add the gateways of the serviced network
+        for (const servicedAccessGatewayId in servicedAccessGateways) {
+          const newServicedAccessGatewayInfo: ServicingAccessGatewayRowType = {
+            networkId: servicedAccessNetwork.id,
+            networkName: servicedAccessNetwork.name,
+            gatewayId: servicedAccessGatewayId,
+            gatewayName: servicedAccessGateways[servicedAccessGatewayId]?.name,
```

Comment:
Can you add access gateway health as well in this table ? @uri200 is there any other fields which will be interesting to look at from federation perspective ?

Reply:
Did that & also updated test file to test that.